tests/PlaywrightWrapper/Program.cs

```diff
@@ -0,0 +1,8 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+//
+sealed class Program
+{
+    public static int Main(string[] args)
+        => Microsoft.Playwright.Program.Main(args);
```

Comment:
Why do we need this wrapper? Why can't we just call Microsoft.Playwright directly?

(It would be good to put the answer in a comment in the code)

Reply:
Looks like creating a simple runtimeconfig.json, and deploying it next to the library is enough to run that directly:

```json
{
  "runtimeOptions": {
    "tfm": "net8.0",
    "framework": {
      "name": "Microsoft.NETCore.App",
      "version": "8.0.0"
    },
    "configProperties": {
      "System.Runtime.Serialization.EnableUnsafeBinaryFormatterSerialization": false
    }
  }
}
```

Reply:
Dropped the project, and directly running the assembly now. It works locally.